devcodeabode/MoDiBo
# Move logs to own directory

**Related bugs/feature requests**



+ Resolves  #6 

**Describe the changes**

Logs now appear in `/logs` instead of `/src`

**Expected behavior**

Logs appear in `/logs`

**Screenshots**


**New npm dependencies**



**Testing Instructions**


1. Start the bot.
2. Verify that `/logs` is created and log files end up there.
3. Stop the bot.
4. `npm run cleanLogs`
5. Verify that the logs are successfully removed.

**Additional context**
